kubectl-cloudflow/deploy/deploy.go

```diff
@@ -92,6 +86,331 @@ func GetCloudflowApplicationDescriptorFromDockerImage(dockerRegistryURL string,
 	return spec, *pulledImage
 }
 
+// CreateApplicationSpecFromBlueprintAndImages pulls all images necessary to create a Cloudflow Application descriptor
+// from a docker label. The function returns the created application spec and the pulled in images
+func CreateApplicationSpecFromBlueprintAndImages(imageRefs []ImageReference,
+	blueprintConfig *configuration.Config, replicas map[string]int) (domain.CloudflowApplicationSpec, []*docker.PulledImage) {
+
+	client, err := docker.GetVersionedClient()
+	if err != nil {
+		util.LogAndExit(err.Error())
```

Comment:
This function should not logAndExit but return an error.

Reply:
done ..